### DISCUSSION TITLE
Documentation of Bootstrap

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [twbs/bootstrap#31825] [Q&A] [answered]
Hello, please how can I get the documentation of `Bootstrap` locally because I don't really get much internet and I use Bootstrap classes alot and some I have to research on

Thank you

**Comment by mdo** [comment, marked as answer]
You can clone the repo and run it locally with our dev scripts in `package.json`. The readme in our project explains this a bit more.

**Reply by oyedejioyewole** [reply]
Thanks
